src/components/Encoding.js

```diff
@@ -75,6 +77,16 @@ const CHANNEL_OPTIONS = [
   }
 ]
 
+const overlayDescriptions = {
```

Comment:
what is the value of having these in this object as opposed to just rendering them within the components? (or passing them directly as arguments to the function i proposed below)

Reply:
Just to make it a bit more cleaner since the descriptions are kind of long - i thought it would be easier to read doing it this way instead of directly passing the descriptions.